functions/src/auto-importer.ts

```diff
@@ -244,7 +256,7 @@ export const monitorSyncDocCount = functions.pubsub.schedule(monitorSyncDocSched
 });
 
 export const syncToS3AfterSyncDocWritten = functions.firestore
-  .document(`${answersSyncPathAllSources}/{runKey}`) // NOTE: {answerId} is a wildcard passed to Firebase
+  .document(`${answersSyncPathAllSources}/{id}`) // NOTE: {answerId} is a wildcard passed to Firebase
```

Comment:
You might want to update this comment: `NOTE: {answerId} is a wildcard passed to Firebase`

Reply:
Sure.

I was looking and couldn't find a way to just say `.document(`${answersSyncPathAllSources}/*`)`, i.e. I don't care about the value of the wildcard. It seems like it might be a tiny bit more efficient, since I don't need it, but I think having something there is a necessity.